components/org.wso2.carbon.identity.api.user.backupcode/org.wso2.carbon.identity.api.user.backupcode.v1/src/gen/java/org/wso2/carbon/identity/rest/api/user/backupcode/v1/MeApiService.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.identity.rest.api.user.backupcode.v1;
+
+import org.wso2.carbon.identity.rest.api.user.backupcode.v1.*;
+import org.wso2.carbon.identity.rest.api.user.backupcode.v1.dto.*;
+
+import org.wso2.carbon.identity.rest.api.user.backupcode.v1.dto.ErrorDTO;
+import org.wso2.carbon.identity.rest.api.user.backupcode.v1.dto.BackupCodeResponseDTO;
+import org.wso2.carbon.identity.rest.api.user.backupcode.v1.dto.UserRequestDTO;
+
+import java.util.List;
+
+import java.io.InputStream;
+import org.apache.cxf.jaxrs.ext.multipart.Attachment;
+
+import javax.ws.rs.core.Response;
+
+public abstract class MeApiService {
```

Comment:
Can we add class level and method level doc comments for this class?

Reply:
These are auto generated class. So IMO we should not change these class

Reply:
Added method and class level comments